pkg/controllers/provisioning/scheduling/nodeclaim.go

```diff
@@ -39,6 +39,7 @@ import (
 // will be turned into one or more actual node instances within the cluster after bin packing.
 type NodeClaim struct {
 	NodeClaimTemplate
+	IsStaticNode bool
```

Comment:
It may be worth seeing if we should store NodeClaims separately in our scheduler rather than storing this bool